base_layer/core/Cargo.toml

```diff
@@ -16,8 +16,11 @@ chrono = "0.4.6"
 tari_infra_derive = { path = "../../infrastructure/derive", version = "0.0.1" }
 digest = "0.8.0"
 tari_crypto = { path = "../../infrastructure/crypto", version = "0.0.1" }
+curve25519-dalek = "1.1.3"
```

Comment:
Why??

We should really abstract this away

Reply:
I actually have no idea why that is there...I don't use it.